src/cljs/test/broadfcuitest/sort_order_submission.cljs

```diff
@@ -0,0 +1,64 @@
+(ns broadfcuitest.sort-order-submission
+  (:require
+   [cljs.test :refer [deftest is testing]]
+   [broadfcui.page.workspace.monitor.common :refer [sort-order-submission]]
+   [broadfcui.page.workspace.monitor.common :refer [sort-order-sub-status]]
+   [broadfcui.page.workspace.monitor.common :refer [sort-order-wf-status]]
+   [broadfcui.utils :as utils]
+   [broadfcui.page.workspace.monitor.common :as moncommon]))
+
+(deftest check-sort-order-submission
+  (let [mixed [{:status "Done" :workflowStatuses {:Aborted 1 :Failed 2 :Succeeded 3}}
+               {:status "Done" :workflowStatuses {:Aborted 2 :Succeeded 3}}
+               {:status "Submitted"}
+               {:status "Aborting" :workflowStatuses {:blah 3}}
+               {:status "Aborted" :workflowStatuses {:Aborted 3}}
+               {:status "Done" :workflowStatuses {:Aborted 2 :Succeeded 3}}
+               {:status "Done" :workflowStatuses {:Aborted 1 :Failed 2 :Succeeded 3}}
+               {:status "Done" :workflowStatuses {:Succeeded 3}}
+               {:status "Submitted"}
+               {:status "Aborting" :workflowStatuses {:blah 3}}
+               {:status "Aborted" :workflowStatuses {:Aborted 1 :Failed 2 :Succeeded 3}}
+               {:status "Done" :workflowStatuses {:Aborted 2 :Succeeded 3}}]
+        expected [{:status "Done" :workflowStatuses {:Aborted 1 :Failed 2 :Succeeded 3}}
+                  {:status "Done" :workflowStatuses {:Aborted 1 :Failed 2 :Succeeded 3}}
+                  {:status "Done" :workflowStatuses {:Aborted 2 :Succeeded 3}}
+                  {:status "Done" :workflowStatuses {:Aborted 2 :Succeeded 3}}
+                  {:status "Done" :workflowStatuses {:Aborted 2 :Succeeded 3}}
+                  {:status "Done" :workflowStatuses {:Succeeded 3}}
+                  {:status "Submitted"}
+                  {:status "Submitted"}
+                  {:status "Aborting" :workflowStatuses {:blah 3}}
+                  {:status "Aborted" :workflowStatuses {:Aborted 3}}
+                  {:status "Aborting" :workflowStatuses {:blah 3}}
+                  {:status "Aborted" :workflowStatuses {:Aborted 1 :Failed 2 :Succeeded 3}}]]
```

Comment:
could you add to this mixed test:
1. a submission status of "Blah"
3. one or more submissions of status "Succeeded"
4. a submission status of "Done" but no workflows
5. a submission status of "Done" and `:workflowStatuses {:blah 3}`

in other words - I don't think this test exercises codes 50, 80, or 90.

Reply:
Adding suggestions 1), 3) and 4). Regarding 2), based on code here: [`ExecutionModel.SubmissionStatuses`](https://github.com/broadinstitute/rawls/blob/c25c8736f96fc020430958a641051bfeda805d54/core/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala#L454) I don't think we'll see a submission status of "Succeeded", so I don't think it makes sense to add it as a test case.